File: uri 1006.java
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args) {
   
    Scanner input = new Scanner(System.in);
        double A=input.nextDouble();
        double B =input.nextDouble();
        double c =input.nextDouble();
        double MEDIA = (A*2+B*3+c*5)/(2.0+3.0+5.0);
        System.out.printf("MEDIA = %.1f\n",MEDIA);
    }
 
}

File: uri 1006.c
#include<stdio.h>
int main()

{
    double A,B,C,MEDIA;
    scanf("%lf%lf%lf",&A,&B,&C);
    A=A*2;
    B=B*3;
    C=C*5;
    MEDIA = (A+B+C)/(2+3+5);
    printf("MEDIA = %0.1lf\n",MEDIA);
    return 0;
}




